src/public/js/index.js

```diff
@@ -0,0 +1,28 @@
+import getTimeRemaining from './countdown';
+
+const initializeClock = (minutes) => {
+  document.getElementById('start').style.display = 'none';
+  document.getElementById('stop').style.display = 'block';
```

Comment:
block is just default for a block level element, we can change it to whatever we want to make it show